src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -31,6 +33,28 @@ public abstract class VariantWalker extends VariantWalkerBase {
     private FeatureDataSource<VariantContext> drivingVariants;
     private FeatureInput<VariantContext> drivingVariantsFeatureInput;
 
+    @Argument(fullName = StandardArgumentDefinitions.VARIANT_OUTPUT_INTERVAL_FILTERING_MODE_LONG_NAME,
```

Comment:
I'm pretty sure all these changes need to go in VariantWalkerBase, not VariantWalker.  I think that's why all the tests exploded.